### ISSUE TITLE
Could NOT find JSONCPP (missing:  JSONCPP_LIBRARY) - OS X 10.10.3

### ISSUE THREAD

**Issue by medoix** [trezor/trezord#13]
Trying to build on OSX 10.10.3 and have used brew to install all other dependencies that were missing and have also installed jsoncpp but it keeps saying its the only one that cannot be found. Any clues on if there is another package it needs but is not telling me about?

```
$ ./build.sh
-- Found libmicrohttpd: /usr/local/lib/libmicrohttpd.dylib
-- Boost version: 1.58.0
-- Found the following Boost libraries:
--   regex
--   thread
--   system
--   unit_test_framework
--   program_options
--   chrono
-- Could NOT find JSONCPP (missing:  JSONCPP_LIBRARY)
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
JSONCPP_LIBRARY
    linked by target "trezord" in directory /Users/medoix/Projects/trezord

-- Configuring incomplete, errors occurred!
See also "/Users/medoix/Projects/trezord/build/CMakeFiles/CMakeOutput.log".
```

**Comment by prusnak** [comment]
Have you issued the `git submodule update --init` command as described in the readme?

**Comment by medoix** [comment]
Yes, before and after i installed the dependencies.

**Comment by snakecharmer1024** [comment]
+1

**Comment by snakecharmer1024** [comment]
Fixed this: if you try running build.sh before the git submodule command, then it creates a folder build/lib/jsoncpp but doesn't put anything in it. However in the build.sh script the sequence that builds jsoncpp only runs if that directory doesn't exist. Not sure exactly the best way to fix this, but deleting the empty jsoncpp folder and then running build.sh makes it work.

rm -r ./build/lib/jsoncpp
./build.sh

**Comment by prusnak** [comment]
Should be fixed in d96796883b806b681ebd600f33c5c967d890b91d.

Thanks for the report and pointing out the problem.
